src/Authoring/DiagnosticTests/NegativeData.cs

```diff
@@ -0,0 +1,2142 @@
+namespace DiagnosticTests
+{
+    public partial class TestDiagnostics
+    {
+
+        // Cases to add -- WIP
+        private const string StructWithInterfaceField = @"
+namespace Test 
+{
+        public interface Foo 
+        {
+            int Id(int i);
+        }
+
+        public struct StructWithIface_Invalid
+        {
+            public Foo ifaceField;
+        }
+}";
+
+        private const string UnsealedClass = @"
+namespace Test 
+{ 
+    public class UnsealedClass 
+    { 
+        public UnsealedClass() {} 
+    } 
+}";
+        private const string UnsealedClass2 = @"
+namespace Test 
+{ 
+    public class UnsealedClass 
+    { 
+        private UnsealedClass() {} 
+    } 
+}"; 
+
+        private const string GenericClass = @"
+namespace Test 
+{ 
+    public sealed class GenericClass<T> 
+    { 
+        public UnsealedClass<T>() {} 
+    } 
+}";
+        private const string GenericInterface = @"
+namespace Test 
+{ 
+    public interface GenIface<T> 
+    { 
+        int Foo(T input); 
+    }
+}";
+
+        private const string InterfaceInheritsException = @"
+namespace Test 
+{ 
+    public interface IfaceWithExceptions : System.Exception 

```

Comment:
Technically this isn't valid C# code.

Reply:
Yeah I thought I had removed this, thanks for catching it